recipes/codan/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0" %}
+
+package:
+  name: CodAn
```

Comment:
I think name needs to be lower case

Reply:
Hi @bernt-matthias. Thank you for your reply. I will change it and re-send the PR.